### PR TITLE
docs: Update next steps after the creation of a new Uno Platform app

### PR DESCRIPTION
**Pull request by agneszitte**
## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## Description
Changes based on the discussion [here](https://teams.microsoft.com/l/message/19:787ad1de1af24252b9041bcbd4ec1ca3@thread.tacv2/1710786271926?tenantId=a297d6c0-b635-41a3-b1e3-558efe71e413&groupId=f55cdec8-036b-4efa-8e1a-c4129ba9bf12&parentMessageId=1710786271926&teamName=Uno%20Platform&channelName=Docs%20and%20Getting%20Started&createdTime=1710786271926)

### CODE REVIEW

**Comment by agneszitte** [comment]
@sasakrsmanovic The changes can be tested here:
https://unodocsprstaging.z13.web.core.windows.net/pr-15969/articles/get-started-wizard.html

**Comment by agneszitte** [comment]
@sasakrsmanovic The changes can be tested here:
[unodocsprstaging.z13.web.core.windows.net/pr-15969/articles/get-started-wizard.html](https://unodocsprstaging.z13.web.core.windows.net/pr-15969/articles/get-started-wizard.html)

**Comment by agneszitte** [comment]
@Mergifyio backport release/stable/5.1

File: doc/articles/get-started-wizard.md [hunk 1]
```diff
@@ -2,19 +2,68 @@
 uid: Uno.GetStarted.Wizard
 ---
 
-# Welcome to Uno Platform
+<!-- markdownlint-disable MD026 -->
+# Done! Important next steps:
 
-**Congratulations, you've just created a new project using the [Uno Platform](https://platform.uno/) project templates!**
+Create a simple cross-platform counter application.  
+Choose one of the 4 counter variants combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).
```
Comment (kazo0) [inline doc/articles/get-started-wizard.md:9]: Maybe capitalize the work `counter` or say "Counter app variants"?

Just seeing "counter variants" makes it sound like something else

**Review by kazo0 — APPROVED** [review]

File: doc/articles/get-started-wizard.md (hunk 1 above)
Comment (sasakrsmanovic) [reply doc/articles/get-started-wizard.md:9]: Agreed with Steve. 

Also, lets please add a newline after this sentence. for better readability  "Create a simple cross-platform counter application."

And also lets capitalize C in Counter in that sentence please: "Create a simple cross-platform Counter application."

**Comment by agneszitte** [comment]
@sasakrsmanovic last verification please:
[unodocsprstaging.z13.web.core.windows.net/pr-15969/articles/get-started-wizard.html](https://unodocsprstaging.z13.web.core.windows.net/pr-15969/articles/get-started-wizard.html)

**Review by sasakrsmanovic — APPROVED** [review]

**Pull request merged**